static/index.scss

```diff
@@ -0,0 +1,17 @@
+@import "./styles/colors";
+
+// @font-face {
+//     font-family: "NotoSansJP";
+//     src: url('static/fonts/NotoSansJP-Regular.ttf') format("truetype");
+// }
+// @mixin NotoSansJP {
```

Comment:
Remove all of these unneeded comments here. Your code that goes to develop branch should never contain commented-out lines of code.